abci.go

```diff
@@ -14,19 +14,19 @@ import (
 // DeliverOrError returns an abci response for DeliverTx,
 // converting the error message if present, or using the successful
 // DeliverResult
-func DeliverOrError(result DeliverResult, err error) abci.ResponseDeliverTx {
+func DeliverOrError(result DeliverResult, err error, debug bool) abci.ResponseDeliverTx {
```

Comment:
Hmm... I had thought of a state variable in this module, but if you can pass it in as an arg, that is cleaner.